test/parallel/test-fs-assert-encoding-error.js

```diff
@@ -0,0 +1,77 @@
+'use strict';
+require('../common');
+const assert = require('assert');
+const fs = require('fs');
+
+const options = 'test';
+const noop = () => {};
+const unknownEncodingMessage =
+  new RegExp(`^Error: Unknown encoding: ${options}$`);
```

Comment:
Since `options` never changes, you can just use `test` in the regex, and use a literal instead of the `RegExp()` constructor.